h2o-algos/src/main/java/hex/glm/GLMMetricBuilder.java

```diff
@@ -223,13 +226,14 @@ protected void computeAIC(){
               metricsBDHGLM._bad, metricsBDHGLM._sumetadiffsquare, metricsBDHGLM._convergence, metricsBDHGLM._randc, 
               metricsBDHGLM._fixef, metricsBDHGLM._ranef, metricsBDHGLM._iterations);
     } else {
-      if (_glmf._family == Family.binomial || _glmf._family == Family.quasibinomial) {
+      if (_glmf._family == Family.binomial || _glmf._family == Family.quasibinomial || 
+              _glmf._family == Family.fractionalbinomial) {
         ModelMetricsBinomial metricsBinommial = (ModelMetricsBinomial) metrics;
         GainsLift gl = null;
         if (preds != null) {
           Vec resp = f.vec(m._parms._response_column);
           Vec weights = f.vec(m._parms._weights_column);
-          if (resp != null) {
+          if (resp != null && !_glmf._family.equals(Family.fractionalbinomial)) { // don't calculate for frac binomial
```

Comment:
```suggestion
          if (resp != null && Family.fractionalbinomial != _glmf._family) { // don't calculate for frac binomial
```

Using == or != results in compile-time safety. Therefore it is preferred. Also, it is more coherent and readable.